<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CDbLoginComponent</class>
 <widget class="QFrame" name="CDbLoginComponent">
  <property name="minimumSize">
   <size>
    <width>300</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>DB login</string>
  </property>
  <property name="frameShape">
   <enum>QFrame::StyledPanel</enum>
  </property>
  <property name="frameShadow">
   <enum>QFrame::Raised</enum>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="2">
    <widget class="QLineEdit" name="le_Username">
     <property name="maxLength">
      <number>40</number>
     </property>
     <property name="placeholderText">
      <string>user name or id</string>
     </property>
     <property name="clearButtonEnabled">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="lbl_SwiftDBIcon">
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap resource="../../blackmisc/blackmisc.qrc">:/own/icons/own/swift/swift32Database.png</pixmap>
     </property>
    </widget>
   </item>
   <item row="3" column="2">
    <widget class="QPushButton" name="pb_Login">
     <property name="text">
      <string>login</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="lbl_Password">
     <property name="text">
      <string>Password:</string>
     </property>
    </widget>
   </item>
   <item row="2" column="2">
    <widget class="QLineEdit" name="le_Password">
     <property name="placeholderText">
      <string>password</string>
     </property>
     <property name="clearButtonEnabled">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="lbl_User">
     <property name="text">
      <string>User:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QLabel" name="lbl_SwiftDB">
     <property name="text">
      <string>DB</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../blackmisc/blackmisc.qrc"/>
 </resources>
 <connections/>
</ui>
